source/includes/gcp/_target_proxies.md

```diff
@@ -42,11 +42,12 @@ Retrieve a list of all target proxies in an [environment](#administration-enviro
 | -------------------------------- | ------ |
 | `creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format. |
 | `id`<br/>*UUID*                  | The id of the backend service. |
-| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy, compute#targetHttpsProxy, compute#targetSslProxy or compute#TargetTcpProxy for target proxies.|
+| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy, compute#targetHttpsProxy, compute#targetSslProxy or compute#TargetTcpProxy for target proxies. |
 | `name`<br/>*string*              | Name of the resource. |
 | `selfLink`<br/>*string*          | Server-defined URL for the resource. |
 | `shortUrlMap`<br/>*string*       | A short version of the URL map. |
-| `type`<br/>*enum*                | A short version of the kind of resource. Either HTTP, HTTPS, SSL or TCP|
+| `shortCertificates`<br/>*string* | A short version of the SSL certificates. |
+| `type`<br/>*enum*                | A short version of the kind of resource. Either HTTP, HTTPS, SSL or TCP. |
```

Comment:
Only mention HTTP/HTTPS, we don't support the others.

Reply:
But the API will return all 4 types?

Reply:
It will, although we could edit the fetcher to only get HTTP/HTTPS.
A customer using cloudmc with the GCP plugin will not have access to the GCP console. So will only be able to see what gets created by cloudmc.

Reply:
ae81aa9d03315427864b7953c7932bce5c8f5420